From e36491854fb169ad4d47dd55b0070381754695d4 Mon Sep 17 00:00:00 2001
From: Beleswar Padhi <b-padhi@ti.com>
Date: Tue, 18 Mar 2025 15:49:30 +0530
Subject: [PATCH 2/5] irqchip: irq-ti-sci-intr: Update Mailbox IRQ input line
 to INTR

The A720 core is configured to be the third user of the mailbox
instance. Accordingly, it has to use the IRQ line 420 which is input to
NAVSS Interrupt Router. Update this IRQ line so that A720 can receive
mailbox interrupts from remote processor.

Signed-off-by: Beleswar Padhi <b-padhi@ti.com>
---
 drivers/irqchip/irq-ti-sci-intr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/irqchip/irq-ti-sci-intr.c b/drivers/irqchip/irq-ti-sci-intr.c
index e75807c24e49..8ea67217f881 100644
--- a/drivers/irqchip/irq-ti-sci-intr.c
+++ b/drivers/irqchip/irq-ti-sci-intr.c
@@ -21,7 +21,7 @@
 #define INTR_OUT_CNT 192
 #define INTR_BCDMA_START 367
 #define INTR_BCDMA_END 318
-#define MAILBOX_INTR 418
+#define MAILBOX_INTR 420
 
 /**
  * struct to distinguish between interrupt router which does not use ti-sci for
-- 
2.34.1

